### PR TITLE
Fix: Use class keyword

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] makes use of the class keyword where possible

Somewhat related to #4907.

### CODE REVIEW

File: tests/unit/Framework/TestCaseTest.php [hunk 1]
```diff
@@ -1083,7 +1086,10 @@ public function testRunBareThrowsExceptionWhenTestHasInvalidName($name): void
         $testCase = new TestWithDifferentNames($name);
 
         $this->expectException(Exception::class);
-        $this->expectExceptionMessage('PHPUnit\Framework\TestCase::$name must be a non-blank string.');
+        $this->expectExceptionMessage(sprintf(
+            '%s::$name must be a non-blank string.',
+            TestCase::class
+        ));
```
Comment (localheinz) [inline tests/unit/Framework/TestCaseTest.php:1092]: @sebastianbergmann 

How do you feel about changes like these?

Comment (sebastianbergmann) [reply tests/unit/Framework/TestCaseTest.php:1092]: Makes sense to me.

Comment (localheinz) [reply tests/unit/Framework/TestCaseTest.php:1092]: @sebastianbergmann 

I'll adjust all the other places, too!

**Comment by sebastianbergmann** [comment]
I cannot merge this to `9.5` without conflicts. Please send a PR for `9.5` should these/similar changes need to be made there. Thanks!

**Comment by localheinz** [comment]
Thank you, @sebastianbergmann, will follow up!

**Pull request merged**